docs/tutorials/index.rst

```diff
@@ -11,4 +11,5 @@ Tutorials
    ../cli.rst
    first_steps_config
    context_manager
-   wrapup
\ No newline at end of file
+   wrapup
```

Comment:
wrapup was included as:

> Jeff Zohrab <jzohrab@gmail.com>
20 April 2016 at 15:42:00 GMT+2
2dedd85b Moved shutdown to end of tutorial.

I modified the index.rst to include a reference to napalm-ansible.rst